src/store/getters.ts

```diff
@@ -10,7 +10,7 @@ import { TreeNode } from '@/types/TreeNode'
 import { Section } from '@/types/Section'
 
 export default {
-  isSignedIn: (state: ApplicationState): boolean => !!state.context && !!state.context.context && state.context.context.authenticated,
+  isSignedIn: (state: ApplicationState): boolean => state.context.context && state.context.context.authenticated,
```

Comment:
why it there context.context

Reply:
the first is the name of the module, the second the name of the context key in the state

Reply:
suggest to refactor this to make ik clear what is going on, why is there a module and a key context

Reply:
yes, I know. Shall we do this in different PR?

Reply:
sure, still this is a refactor PR ?